assets/admin/editor-wizard/steps/course-patterns-step.js

```diff
@@ -25,6 +29,10 @@ const CoursePatternsStep = ( { data, ...props } ) => {
 		replaces[ 'sensei-content-description' ] = data.courseDescription;
 	}
 
+	if ( user.name ) {
+		replaces[ 'sensei-content-author' ] = `<strong>${ user.name }</strong>`;
```

Comment:
Maybe it would be nice to make it uppercase the first letter at least... but very minor thing and maybe not something everybody likes so feel free to ignore this :D

Reply:
I think we could keep it, so the users can select in the profile what's their display name to the site. Just in case for any reason someone prefers to use lowercase, so they have the freedom to choose. 🤷‍♂️